deployment/src/strongmind_deployment/route53_zone.py

```diff
@@ -0,0 +1,77 @@
+from enum import Enum
+from typing import Optional, Dict
+import pulumi
+import pulumi_aws as aws
+from pydantic import BaseModel
+
+
+class Route53ZoneArgs(BaseModel):
+    domain_name: str
+    vpc_id: Optional[str] = None
+    vpc_region: Optional[str] = None
+    comment: Optional[str] = "Managed by Pulumi"
+    force_destroy: Optional[bool] = False
+    delegation_set_id: Optional[str] = None
+    tags: Optional[Dict[str, str]] = None
+
+
```

Comment:
I probably named this Enum in the old code.   They aren't outputs though.   So it's not an accurate name.

Presumably these are used in the consumer code to set exports in shared_infrastructure.

maybe a better name is `R53ZoneExportNames`.